hydromt_wflow/wflow.py

```diff
@@ -3163,14 +3171,22 @@ def read_staticgeoms(
     def write_geoms(
         self,
         geom_fn: str = "staticgeoms",
+        precision: int = 5,
```

Comment:
I thought we said we would change default precision to 6?

Reply:
And maybe in the code, you could check is the mod.crs.is_projected and then adjust the default value to 1 for 0.1 meter? And document it in the docstring?
Which I now see should maybe be updated?